.github/workflows/docs-issue.yml

```diff
@@ -37,5 +37,6 @@ jobs:
     with:
         issue_repository: "dbt-labs/docs.getdbt.com"
         issue_title: "[Core] Docs Changes Needed from ${{ github.event.repository.name }} Issue #${{ github.event.issue.number }}"
-        issue_body: "At a minimum, update body to include a link to the page on docs.getdbt.com requiring updates and what part(s) of the page you would like to see updated."
+        issue_body: "At a minimum, update body to include a link to the page on docs.getdbt.com requiring updates and what part(s) of the page you would like to see updated.\n Originating from this issue: https://github.com/dbt-labs/dbt-core/issues/${{ github.event.issue.number }}"
+        label: 'dbt-core v1.9'
```

Comment:
Since we're opening an issue in another repo we can't actually add any labels.  The `community-label.yml` workflow works entirely within the core repo which is why it can add labels to issues.

Right now you're adding labels to these autogenerated issues within your own repo with [autogenerated_labeler.yml](https://github.com/dbt-labs/docs.getdbt.com/blob/current/.github/workflows/autogenerated_labeler.yml).

Reply:
i see. thank you for the info and extra context